blocks/index.js

```diff
@@ -8,5 +8,3 @@
 // Blocks are inferred from the HTML source of a post through a parsing mechanism
 // and then stored as objects in state, from which it is then rendered for editing.
 export * from './api';
-
```

Comment:
@youknowriad - did you update this comment above in the PR where you migrate blocks to store? :)

Reply:
haha :) comment posted on the wrong PR